src/commands/project/pipeline.ts

```diff
@@ -148,14 +149,14 @@ const createPipeline = async (
   devopsClient: IBuildApi
 ): Promise<BuildDefinition> => {
   const definition = definitionForAzureRepoPipeline({
-    branchFilters: ["master"], // I believe this is intentional. Our pipelines are triggered only by merges into the master branch.
+    branchFilters: ["master"], // Pipelines are triggered only by merges into the master branch.
```

Comment:
_only_ the reconcile pipeline/project pipeline, service build pipeline is triggered on ring branches